docs/building-blocks/extending-templates/sequential-loop.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Extending Azure Resource Manager template functionality - sequential looping
+description: Describes how to extend the functionality of Azure Resource Manager templates to implement sequential looping when deploying multiple instances of a resource type.
+author: petertay
+ms.service: guidance
+ms.topic: article
+ms.date: 05/03/2017
+ms.author: pnp
+
+---
+
+# Azure Resource Manager templates - sequential looping
```

Comment:
Title something like "Use sequential looping in Azure Resource Manager templates"? The hyphen looks kind of odd here.